Large AOI Causing Excel Export Freakout

This may be a red herring to why the reports are failing online but when I run the attached geojson file I get this error:

[royal_gorge.geojson.zip](https://github.com/user-attachments/files/23781557/royal_gorge.geojson.zip)

We probably need to skip excel generation when we're running large geometries

```log
[INFO] [RSDataFrame] Dropping geometry columns for Excel export: ['dgo_polygon_geom']
[ERROR] [Setup] At least one sheet must be visible
Traceback (most recent call last):
  File "/Users/matt/Work/Git/Riverscapes/rs-reports-gen/src/util/pandas/RSGeoDataFrame.py", line 100, in export_excel
    baked_gdf.to_excel(writer, sheet_name="data", index=False)
  File "/Users/matt/Work/Git/Riverscapes/rs-reports-gen/.venv/lib/python3.12/site-packages/pandas/util/_decorators.py", line 333, in wrapper
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/Users/matt/Work/Git/Riverscapes/rs-reports-gen/.venv/lib/python3.12/site-packages/pandas/core/generic.py", line 2439, in to_excel
    formatter.write(
  File "/Users/matt/Work/Git/Riverscapes/rs-reports-gen/.venv/lib/python3.12/site-packages/pandas/io/formats/excel.py", line 931, in write
    raise ValueError(
ValueError: This sheet is too large! Your sheet size is: 1273033, 57 Max sheet size is: 1048576, 16384

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/matt/Work/Git/Riverscapes/rs-reports-gen/src/reports/rpt_rivers_need_space/main.py", line 240, in main
    make_report_orchestrator(
  File "/Users/matt/Work/Git/Riverscapes/rs-reports-gen/src/reports/rpt_rivers_need_space/main.py", line 191, in make_report_orchestrator
    RSGeoDataFrame(data_gdf).export_excel(os.path.join(report_dir, 'data', 'data.xlsx'))
  File "/Users/matt/Work/Git/Riverscapes/rs-reports-gen/src/util/pandas/RSGeoDataFrame.py", line 99, in export_excel
    with pd.ExcelWriter(output_path) as writer:
  File "/Users/matt/Work/Git/Riverscapes/rs-reports-gen/.venv/lib/python3.12/site-packages/pandas/io/excel/_base.py", line 1353, in __exit__
    self.close()
  File "/Users/matt/Work/Git/Riverscapes/rs-reports-gen/.venv/lib/python3.12/site-packages/pandas/io/excel/_base.py", line 1357, in close
    self._save()
  File "/Users/matt/Work/Git/Riverscapes/rs-reports-gen/.venv/lib/python3.12/site-packages/pandas/io/excel/_openpyxl.py", line 110, in _save
    self.book.save(self._handles.handle)
  File "/Users/matt/Work/Git/Riverscapes/rs-reports-gen/.venv/lib/python3.12/site-packages/openpyxl/workbook/workbook.py", line 386, in save
    save_workbook(self, filename)
  File "/Users/matt/Work/Git/Riverscapes/rs-reports-gen/.venv/lib/python3.12/site-packages/openpyxl/writer/excel.py", line 294, in save_workbook
    writer.save()
  File "/Users/matt/Work/Git/Riverscapes/rs-reports-gen/.venv/lib/python3.12/site-packages/openpyxl/writer/excel.py", line 275, in save
    self.write_data()
  File "/Users/matt/Work/Git/Riverscapes/rs-reports-gen/.venv/lib/python3.12/site-packages/openpyxl/writer/excel.py", line 89, in write_data
    archive.writestr(ARC_WORKBOOK, writer.write())
                                   ^^^^^^^^^^^^^^
  File "/Users/matt/Work/Git/Riverscapes/rs-reports-gen/.venv/lib/python3.12/site-packages/openpyxl/workbook/_writer.py", line 150, in write
    self.write_views()
  File "/Users/matt/Work/Git/Riverscapes/rs-reports-gen/.venv/lib/python3.12/site-packages/openpyxl/workbook/_writer.py", line 137, in write_views
    active = get_active_sheet(self.wb)
             ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/matt/Work/Git/Riverscapes/rs-reports-gen/.venv/lib/python3.12/site-packages/openpyxl/workbook/_writer.py", line 35, in get_active_sheet
    raise IndexError("At least one sheet must be visible")
IndexError: At least one sheet must be visible
Backend macosx is interactive backend. Turning interactive mode on.

```

--- a/src/reports/rpt_stream_names/main.py
+++ b/src/reports/rpt_stream_names/main.py
@@ -126,7 +126,7 @@ def make_report_orchestrator(report_name: str, report_dir: Path, path_to_shape: 
         log.info("Querying athena for data for AOI")
         data_df = get_wcdata_for_aoi(query_gdf)
         data_df.to_csv(csv_data_path)
-
+    # given the data groups by stream name and there are only ~80k distinct stream names in CONUS this shouldn't blow up
     data_df.to_excel(report_dir / 'data' / 'data.xlsx', index=False)
 
     # make html report

--- a/src/util/pandas/RSGeoDataFrame.py
+++ b/src/util/pandas/RSGeoDataFrame.py
@@ -70,8 +70,10 @@ class RSGeoDataFrame(gpd.GeoDataFrame):
 
         Args:
             output_path (str): The path to save the Excel file.
-        extra debugs added as this seems to be taking long time
-        todo: optimize and remove them
+
+        * will not export if size of dataframe exceeds limit
+        * extra debugs added as this seems to be taking long time
+        * Future Enhancement: optimize performance and remove extra debugs
         """
         self.log.info(f"Exporting data to Excel at {output_path}")
 

